frontend/src/metabase/parameters/utils/dashboards.ts

```diff
@@ -228,25 +228,3 @@ export function getFilteringParameterValuesMap(
 
   return filteringParameterValues;
 }
-
-export function getParameterValuesSearchKey({
```

Comment:
Since we clear the cache each time the dashboard is updated (like before), this has no meaning